bin/plugin-check.sh

```
@@ -0,0 +1,109 @@
+	echo "error: wp-cli is required and was not found on PATH." >&2
+	exit 1
+}
+
+# wp-content/plugins is this checkout's parent.
+PLUGINS_DIR="$(cd "$ROOT/.." && pwd)"
+STAGE="$ROOT/build/$SLUG"
+TARGET="$PLUGINS_DIR/$TMP_SLUG"
+
+# Always remove the temporary copy, however this script ends. Leaving a second
+# copy of the plugin in wp-content/plugins would be worse than any lint finding.
+cleanup() { rm -rf "$TARGET"; }
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)